lms/models/lti_params.py

```diff
@@ -169,6 +169,11 @@ def _to_lti_v11(v13_params):
     if "roles" in v11_params:
         context_roles = []
         for role in v11_params["roles"]:
+            # Consider administrator regardless of their scope
+            if "administrator" in role.lower():
+                context_roles.append(role)
+                continue
```

Comment:
I wonder if we want to be conservative and only match specific strings like `"http://purl.imsglobal.org/vocab/lis/v2/institution/person#Administrator"` here rather than any string with `"administrator"` anywhere in it?

Reply:
I've seen these values in the DB:

```
Administrator
http://purl.imsglobal.org/vocab/lis/v2/membership#Administrator
http://purl.imsglobal.org/vocab/lis/v2/institution/person#Administrator
http://purl.imsglobal.org/vocab/lis/v2/system/person#Administrator
```

I switched it to `endswith`